### PR TITLE
Typography

### PR DESCRIPTION
**Pull request by josuebran**
### Typography
- Updates color, font weight and line height of docs, breadcrumbs and toc menu
- Updates font size with css calc method used in antora. Please let me know if we should use another method.

### Formatting Headers and titles
- Updates padding and margin for headers and title
- Updates padding and margin for body, article, toc sidebar and breadcrumb

Demo: https://docs-ui-typography.netlify.app/docs/

### CODE REVIEW

File: src/css/vars.css [hunk 1]
```diff
@@ -31,16 +31,16 @@
...
   --nav-muted-color: var(--ds-text-tertiary);
   --nav-panel-divider-color: var(--ds-divider);
   --nav-secondary-background: var(--ds-background-body);
   /* toolbar */
   --toolbar-background: var(--ds-background-body);
   --toolbar-border-color: var(--ds-divider);
-  --toolbar-font-color: var(--ds-text-primary);
-  --toolbar-muted-color: var(--ds-text-disabled);
+  --toolbar-font-color: var(--ds-primary-main);
+  --toolbar-muted-color: var(--ds-text-tertiary);
```
Comment (colegoldsmith) [inline src/css/vars.css:43]: Nice change using the tokens! Lol I'm not sure why I set that to the disabled color

**Review by colegoldsmith — APPROVED** [review]
Looks great! Nice work

**Comment by josuebran** [comment]
@colegoldsmith Yes, [postcss-mixins](https://github.com/postcss/postcss-mixins) would be helpful

**Comment by colegoldsmith** [comment]
@josuebran Do you want to add postcss mixins to the gulp build and make the change in this PR? Otherwise you can just merge this and refactor once we get the plugin added later

**Pull request merged**